pkg/csi/controller_server.go

```diff
@@ -25,27 +30,33 @@ const (
 	timeoutAttachDetach = 120 * time.Second
 	tickAttachDetach    = 2 * time.Second
 	paramHostSC         = "hostStorageClass"
+	LHVolumeNS          = "longhorn-system"
+	HARVNS              = "harvester-system"
```

Comment:
Can we use HarvesterNamespace and LonghornNamespace for the constant name for readibility.

Reply:
Sure, Update Var to LonghornNS and HarvesterNS